var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -79,6 +74,42 @@ class QuantumEspresso(CMakePackage):
         conflicts("^openblas threads=none")
         conflicts("^openblas threads=pthreads")
 
+    # Add cuda support 
+    with when('+cuda'):
+        # only nvhpcsdk compiler is supported
+        conflicts("%aocc")
+        conflicts("%apple_clang")
+        conflicts("%arm")
+        conflicts("%cce")
+        conflicts("%clang")
+        conflicts("%fj")
+        conflicts("%gcc")
+        conflicts("%intel")
+        conflicts("%nag")
+        conflicts("%oneapi")
+        conflicts("%xl")
+        conflicts("%xl_r")
```

Comment:
Can we enable `+cuda` variant only when NVHPC is used instead of having this long list of conflicts.

Reply:
+cuda variant is now enabled only when NVHPC is used b846a23f487af9289e4274e530a7b4715edbbe05